Examples/PcapPlusPlus-benchmark/README.md

```diff
@@ -6,3 +6,12 @@ This is a benchmark application used for measuring PcapPlusPlus performance. It
 See this page for more details: https://pcapplusplus.github.io/docs/benchmark
 
 This application currently compiles on Linux only (where benchmark was running on)
+
+Additionally, there is an integration of Google Benchmark library for benchmarking. For enabling, configure the library with `-DPCAPPP_USE_GOOGLEBENCHMARK=ON`. Each benchmark can be affected from different sources. You can check the table below for more information. If you have performance critical applications which will use PcapPlusPlus, please benchmark it in your environment to get better and more precise results. Also using different pcaps, both in terms of size (larger is better to prevent open/close overhead) and protocol/session variety, can also help to give idea about performance of PcapPlusPlus in your environment.
```

Comment:
I used ChatGPT to find better wording for "Each benchmark can be affected from different sources" and it suggested: "Each benchmark can be influenced by various factors"

Reply:
Check 557839a